src/domain/common/memo/conversion/yjs.state.to.markdown.ts

```
@@ -48,9 +48,63 @@ export const yjsStateToMarkdown = (state: Buffer) => {
+    node: ProseMirrorNode,
+    depth = 0,
+    parentType = ''
+  ): string => {
+    // Ordered lists need 4 spaces per level, bullet lists need 2
+    const indentSize = parentType === 'orderedList' ? 4 : 2;
+    const indent = ' '.repeat(depth * indentSize);
+
+    switch (node.type.name) {
+      case 'bulletList':
+      case 'orderedList':
+        let listOutput = '';
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)